sources/US/MI/hunter-access-2017.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
There's an error processing this file, but i dont think it's your fault.

Reply:
Do I need to change the source url?

Reply:
No, it’s some sort of error with the software that reads these files, I’ll fix it, but I can’t merge this until I do.